components/browser/engine-gecko-nightly/src/test/java/mozilla/components/test/ReflectionUtils.kt

```diff
@@ -0,0 +1,16 @@
+package mozilla.components.test
```

Comment:
nit: License header

Reply:
We need a lint :) Fixed, thanks for the review!